app/dashboard/migrations/0150_auto_20200904_1505.py

```diff
@@ -8,21 +8,21 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('ptokens', '0001_initial'),
+        # ('ptokens', '0001_initial'),
         ('dashboard', '0149_add_twitter_verify_status'),
     ]
 
     operations = [
-        migrations.AddField(
-            model_name='activity',
-            name='ptoken',
-            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='ptoken_activities', to='ptokens.PersonalToken'),
-        ),
-        migrations.AddField(
-            model_name='activity',
-            name='redemption',
-            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to='ptokens.RedemptionToken'),
-        ),
+        # migrations.AddField(
+        #     model_name='activity',
+        #     name='ptoken',
+        #     field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='ptoken_activities', to='ptokens.PersonalToken'),
+        # ),
+        # migrations.AddField(
+        #     model_name='activity',
+        #     name='redemption',
+        #     field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to='ptokens.RedemptionToken'),
+        # ),
```

Comment:
Is there a reason to keep this around? I'd prefer to delete unused code over commenting it out and leaving it in. We can always pull this back from git if it's needed.

Reply:
So this is only cause it's a migration which are auto generated ! 
Ideally we wouldn't want to change this but when we remove a django it causes travis to complain as it runs through the boot